tests/db-utils.ts

```diff
@@ -1,16 +1,17 @@
 import { faker } from '@faker-js/faker'
 import bcrypt from 'bcryptjs'
-import { memoizeUnique } from './memoize-unique.ts'
-
-const unique = memoizeUnique(faker.internet.userName)
+import { UniqueEnforcer } from 'enforce-unique'
 
 export function createUser() {
 	const firstName = faker.person.firstName()
 	const lastName = faker.person.lastName()
 
-	const username = unique({
-		firstName: firstName.toLowerCase(),
-		lastName: lastName.toLowerCase(),
+	const uniqueUsernameEnforcer = new UniqueEnforcer()
```

Comment:
If I understand the library properly, we need to create this object outside the createUser function.

Reply:
Changed